common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2109,12 +2109,6 @@
     ],
     "sqlState" : "F0000"
   },
-  "INVALID_EMPTY_LOCATION" : {
```

Comment:
I'd like to keep this error condition but add 4 sub-classes: TABLE, SCHEMA, PARTITION, DIRECTORY.